.travis.yml

```diff
@@ -3,7 +3,8 @@ cache: bundler
 sudo: required
 env:
   - TEST_GROUP="./script/check_syntax"
-  - TEST_GROUP="rspec spec"
+  - TEST_GROUP="rspec spec --tag ~rspec_run_two"
+  - TEST_GROUP="rspec spec --tag  rspec_run_two"
```

Comment:
We discussed this in chat, but I think splitting the Rspec tests into two runs because of a conflict between the StoryParser tests and a single call to `fake_login` method in the WorksController test feels like overkill. 

As far as I can tell, the StoryParser test is failing because the StoryParser internally calls `User.current_user` and then attempts to get that user's default Pseud, but the `fake_login` method has stubbed `User.current_user` to return a user with no default pseud.

However, I have some good news! The behaviour for restricted works is already covered in https://github.com/otwcode/otwarchive/blob/master/features/works/work_lock.feature#L9-L49 so you can fix this problem by removing the test on line 106 of spec/controllers/works_controller_spec.rb altogether and the problem will go away. 🎉